awx/api/serializers.py

```diff
@@ -643,13 +643,14 @@ def to_representation(self, obj):
 
 class UnifiedJobSerializer(BaseSerializer):
     show_capabilities = ['start', 'delete']
+    events_processed = serializers.SerializerMethodField()
```

Comment:
Can you make this Boolean so options is accurate?

Reply:
I was just googling for this... I don't know how. If I make a boolean field, then DRF expectation is that it's present on the model.

I guess that forces our hand, and we should just cave and add it as a model property?